user.go

```diff
@@ -105,3 +111,18 @@ func (u *User) AvatarURL(size string) string {
 	}
 	return URL
 }
+
+// BannerURL returns a URL to the user's banner.
+// An empty string is returned if the user banner is unset.
+func (u *User) BannerURL() string {
+	var URL string
+	if u.Banner == "" {
+		URL = ""
+	} else if strings.HasPrefix(u.Banner, "a_") {
+		URL = EndpointUserBannerAnimated(u.ID, u.Avatar)
+	} else {
+		URL = EndpointUserBanner(u.ID, u.Banner)
+	}
+
+	return URL
+}
```

Comment:
You're now trying to use a `*string` as a `string` in this function. I would also recommend adding a check for `u.Banner == nil` here before dereferencing it such that you stay consistent with the comment of "An empty string is returned if the user banner is unset."; unset might correspond to the case of `null`/`nil`.